apps/prairielearn/src/pages/instructorStudents/instructorStudents.html.tsx

```diff
@@ -0,0 +1,224 @@
+    () => [
+      columnHelper.accessor('user.uid', {
+        header: 'UID',
+        cell: (info) => info.getValue(),
+      }),
+      columnHelper.accessor('user.name', {
+        header: 'Name',
+        cell: (info) => info.getValue() || '—',
+      }),
+      columnHelper.accessor('user.email', {
+        header: 'Email',
+        cell: (info) => info.getValue() || '—',
+      }),
+      columnHelper.accessor('enrollment.created_at', {
+        header: `Enrolled on (${formatTz(timezone)})`,
+        cell: (info) => {
+          const date = new Date(info.getValue());
+          return (
+            <div style={{ fontVariantNumeric: 'tabular-nums' }}>
+              {formatDate(date, timezone, {
+                includeTz: false,
+              })}
+            </div>
+          );
+        },
+      }),
+    ],
+    [],
+  );
+
+  const allColumnIds = columns
+    .filter((col) => col.accessorKey)
+    .map((col) => col.accessorKey.replace('.', '_'));
+  const [columnVisibility, setColumnVisibility] = useQueryState(
+    'columns',
+    parseAsColumnVisibilityStateWithColumns(allColumnIds).withDefault(
+      Object.fromEntries(allColumnIds.map((id) => [id, true])),
+    ),
+  );
+
+  const table = useReactTable({
+    data: students,
+    columns,
+    columnResizeMode: 'onChange',
+    getRowId: (row) => row.user.user_id,
+    state: {
+      sorting,
+      columnFilters,
+      globalFilter,
+      columnSizing,
+      columnVisibility,
+      columnPinning,
+    },
+    onSortingChange: setSorting,
+    onColumnFiltersChange: setColumnFilters,
+    onGlobalFilterChange: setGlobalFilter,
+    onColumnSizingChange: setColumnSizing,
+    onColumnVisibilityChange: setColumnVisibility,
+    onColumnPinningChange: setColumnPinning,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    getFilteredRowModel: getFilteredRowModel(),
+    defaultColumn: {
+      minSize: 150,
+      size: 300,
+      maxSize: 500,
+      enableSorting: true,
+      enableHiding: true,
+      enablePinning: true,
+    },
+  });
+
+  return (
+    <div class="card d-flex flex-column h-100">
+      <div class="card-header bg-primary text-white">
+        <div class="d-flex align-items-center justify-content-between">
+          <div>Students</div>
+          <div>
+            <DownloadButton
+              students={students}
+              course={course}
+              courseInstance={courseInstance}
+              table={table}
+            />
+          </div>
+        </div>
+      </div>
+      <div class="card-body d-flex flex-column">
+        <div class="d-flex flex-row mb-2">
+          <div class="col-xl-4 col-md-6 col-12 col-auto">
+            <input
+              ref={searchInputRef}
+              type="text"
+              class="form-control"
+              aria-label="Search by UID, name or email."
+              placeholder="Search by UID, name, email..."
+              value={globalFilter}
```

Comment:
For the search box, can we get a clear button something like the x on the right side here?
<img width="233" alt="image" src="https://github.com/user-attachments/assets/b56114c3-23b9-4ff5-915b-c3da21a87a29" />